test/.stats-app/image.js

```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image'
+import logo from './nextjs.png'
+
+function ImagePage(props) {
+  return (
+    <>
+      <h1>next/image example</h1>
+      <Image src={logo} placeholder="blur" />
```

Comment:
Do we also want a couple other usages, such as blur and external image?

Reply:
If either of these could impact the resulting client bundle differently it would be good to add these as well yeah, potentially in separate pages so we can see the impact separately

Reply:
Hmm, I guess it won't affect the client bundle because all of the code for the image component lives in a single file.

The only thing that might affect the client bundle is config in `next.config.js` when that gets serialized to JSON, but thats a 1-to-1 mapping so we don't really need to test it 👍